projects/ng-components/lib/toggle-selector/toggle-selector.component.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { ToggleSelectorComponent, IToggleSelector } from "./toggle-selector.component";
+import { Component, DebugElement, ViewChild } from "@angular/core";
+import { FormsModule } from "@angular/forms";
+import { CommonModule } from "@angular/common";
+import { By } from "@angular/platform-browser";
+
+@Component({
+    selector: "test-sebng-toggle-selector",
+    template: `<sebng-toggle-selector
+        #toggleSelector
+        [multi]="multi"
+        [list]="list"
+        [(ngModel)]="model"
+        [disabled]="disabled"
+        [error]="error"
+        [errorMessage]="errorMessage"
+    ></sebng-toggle-selector>`,
+})
+class ToggleSelectorTestComponent {
+    @ViewChild("toggleSelector") toggleSelector: ToggleSelectorComponent;
+    list: Array<IToggleSelector>;
+    multi: boolean = false;
+    disabled?: boolean = false;
+    model?: any;
+    error: boolean;
+    errorMessage: string;
+
+    constructor() {
+        this.list = [
+            { value: "1", label: "Four mississipi", disabled: true },
+            { value: "2", label: "Five mississipi" },
+        ];
+    }
+}
```

Comment:
I think if you extend `ToggleSelectorComponent`, then you won't need to redfined the props. Can you try that?
```suggestion
class ToggleSelectorTestComponent extends ToggleSelectorComponent {
    @ViewChild("toggleSelector") toggleSelector: ToggleSelectorComponent;

    constructor() {
        this.multi = this.disabled = false;
        this.list = [
            { value: "1", label: "Four mississipi", disabled: true },
            { value: "2", label: "Five mississipi" },
        ];
    }
}
```

Reply:
Im not redefining the Inputs of the components, those properties are variables that needs to be passed to the test comoponent and they are useful for passing different values through-out the test.

```typescript
@Component({
    selector: "test-sebng-toggle-selector",
    template: `<sebng-toggle-selector
        #toggleSelector
        [multi]="multi"
        [list]="list"
        [(ngModel)]="model"
        [disabled]="disabled"
        [error]="error"
        [errorMessage]="errorMessage"
    ></sebng-toggle-selector>`,
})
```

Reply:
Same usage, if you extend it, it will already be there. I use that technique a lot. Try it la